langlib/lang.xml/src/main/ballerina/src/lang.xml/xml.bal

```diff
@@ -154,3 +154,8 @@ public function appendChildren(xml x,xml children) = external;
 # + x - The xml source
 # + qname - Namespace qualified name of the children to be removed
 public function removeChildren(xml x,string qname) = external;
+
+// Functional programming methods
+public function map(xml x, function(xml|string item) returns xml|string func) returns xml = external;
```

Comment:
Do we need to put doc comments for these? @hasithaa any idea? these are already in the spec repo

Reply:
Yes, We need to fix all langlib modules, with proper doc comments.